SSA.lean

```diff
@@ -4,8 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 
 -- Core
 -- ====
-import SSA.Core.Framework
-
+import SSA.Core
```

Comment:
This and the next two changes seem a general refactor that is worth committing separately, no?

Reply:
I feel like this relates to the PR vs commit thing. Absolutely, it should be a different commit, but if PRs and commits are separate things then it's a non-issue.

Reply:
@bollu kindly factored this out.